src/asciidoc/reference/packages/withkojiDatabase.adoc

```diff
@@ -8,9 +8,6 @@ The @withkoji/database package enables you to
 implement a Koji database for the backend of your template.
 //end::description[]
 
```

Comment:
See https://github.com/madewithkoji/docs/pull/71#discussion_r494673014

Reply:
Same thing here... A sentence or two about the structure, since it is prescribed, would be good to know. That info would allow a developer to understand how these methods work and evaluate whether this is the right package for them.